ui/admin/tests/acceptance/roles/scopes-test.js

```diff
@@ -93,4 +104,71 @@ module('Acceptance | roles | scopes', function (hooks) {
     await visit(urls.roleScopes);
     assert.dom('.role-grant-scopes div div:nth-child(3) a').doesNotExist();
   });
+
+  test('toggle and save scope keywords to add', async function (assert) {
+    instances.role.update({ grant_scope_ids: [] });
+    await visit(urls.role);
+
+    await click(`[href="${urls.roleScopes}"]`);
+
+    assert.strictEqual(findAll(TABLE_ROW_SELECTOR).length, 0);
+
+    await click(MANAGE_SCOPES_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.manageScopes);
+
+    // Click three times to select, unselect, then reselect (for coverage)
```

Comment:
Why exactly are we clicking it three times?

Reply:
saw it was a pattern in other similar test cases to cover both toggle on and toggle off